automation/run-tests.sh

```diff
@@ -53,8 +53,9 @@ function add_extra_networks {
     '
 }
 
-function dump_network_info {
+function show_system_info {
     docker_exec '
+      rpm -q NetworkManager;
```

Comment:
Placing this here is a bit strange because we do not need to show this info more than once.
The others are showed before and after the run.

Any chance you will place it outside?

Reply:
moved it outside and changed it to also show the OS version